RELEASE_NOTES.md

```diff
@@ -120,22 +122,22 @@ accompanying blog post.
 
 - Try to get color-space for PNG file from the cICP chunk. This was
   added in a recent revision of the PNG spec, so we take advantage of
-  it is present.
+  it if present.
```

Comment:
comma before the **if**?

Reply:
You seem to like commata a lot. From my grammar learned a long time ago, not here neither at some places above.

Reply:
Sure, I'm not a native speaker. Some of those commas came from a checker, LanguageTool. :)

Reply:
I don't know LanguageTool. I personally find Deepl one of the best one.

Reply:
DeepL is a translator, is it not?
[Grammarly](https://www.grammarly.com/) has a free offer that allows quite a few checks. It supports only English.
[LanguageTool](https://languagetool.org/) is based on open-source tools, with some extra paid components. I have a subscription, as it supports German as well as English, and German is what I'm currently learning.

Reply:
Comma would be ok here but not absolutely necessary. Think of it as a less severe form of bracketing. If you can bracket the whole thing off - "we take advantage of it (if present)" - and the sentence still makes sense then a comma could be useful. Here it's optional I think.